From ba098aea0a7c836b14c094bbac4280baacd874f6 Mon Sep 17 00:00:00 2001
From: Eric Ruei <e-ruei1@ti.com>
Date: Thu, 15 Feb 2018 18:05:09 -0500
Subject: [PATCH 162/176] Revert "arm: dts: am571x-idk: add pruss_eth
 sync/latch control pins"

This reverts commit 0685785ec4446478c09283dd3ade0031f9ece077.
---
 arch/arm/boot/dts/am571x-idk.dts         | 27 ---------------------------
 arch/arm/boot/dts/am57xx-idk-common.dtsi |  2 +-
 2 files changed, 1 insertion(+), 28 deletions(-)

diff --git a/arch/arm/boot/dts/am571x-idk.dts b/arch/arm/boot/dts/am571x-idk.dts
index 7cefcd3..4bf9214 100644
--- a/arch/arm/boot/dts/am571x-idk.dts
+++ b/arch/arm/boot/dts/am571x-idk.dts
@@ -298,16 +298,6 @@
 	pinctrl-5 = <&pps1_timer15_latch_off>;
 };
 
-&pruss1_eth {
-	pinctrl-names = "sync_latch";
-	pinctrl-0 = <&pr1_edc_sync_latch>;
-};
-
-&pruss2_eth {
-	pinctrl-names = "sync_latch";
-	pinctrl-0 = <&pr2_edc_sync_latch>;
-};
-
 &dra7_pmx_core {
 	pps1_timer16_pwm_on: pps1_timer16_pwm_on {
 		pinctrl-single,pins = <
@@ -345,23 +335,6 @@
 		>;
 	};
 
-	pr1_edc_sync_latch: pr1_edc_sync_latch {
-		pinctrl-single,pins = <
-			/* pr1/sync0 */
-			DRA7XX_CORE_IOPAD(0x3724, PIN_OUTPUT | MUX_MODE13)
-			/* pr1/latch0 */
-			DRA7XX_CORE_IOPAD(0x3694, PIN_INPUT  | MUX_MODE12)
-		>;
-	};
-
-	pr2_edc_sync_latch: pr2_edc_sync_latch {
-		pinctrl-single,pins = <
-			/* pr2/sync0 */
-			DRA7XX_CORE_IOPAD(0x3728, PIN_OUTPUT | MUX_MODE13)
-			/* pr2/latch0 */
-			DRA7XX_CORE_IOPAD(0x3698, PIN_INPUT  | MUX_MODE12)
-		>;
-	};
 };
 
 &ptp_bc {
diff --git a/arch/arm/boot/dts/am57xx-idk-common.dtsi b/arch/arm/boot/dts/am57xx-idk-common.dtsi
index 04d1766..6678d70 100644
--- a/arch/arm/boot/dts/am57xx-idk-common.dtsi
+++ b/arch/arm/boot/dts/am57xx-idk-common.dtsi
@@ -158,7 +158,7 @@
 	};
 
 	/* Dual-MAC Ethernet application node on PRU-ICSS2 */
-	pruss2_eth: pruss2_eth {
+	pruss2_eth {
 		compatible = "ti,am57-prueth";
 		pruss = <&pruss2>;
 		sram = <&ocmcram1>;
-- 
1.9.1

